pkg/reconciler/pipelinerun/resources/pipelinespec.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2019 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resources
+
+import (
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+	"golang.org/x/xerrors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// GetPipeline is a function used to retrieve Pipelines.
+type GetPipeline func(string) (v1alpha1.PipelineInterface, error)
+type GetPipelineRun func(string) (*v1alpha1.PipelineRun, error)
```

Comment:
Is this used anywhere?  I can't find any other reference to it.

Reply:
nope, deleted.